proto/prysm/v1alpha1/cloners.go

```diff
@@ -512,3 +512,53 @@ func CopyBlindedBeaconBlockBodyBellatrix(body *BlindedBeaconBlockBodyBellatrix)
 		ExecutionPayloadHeader: CopyExecutionPayloadHeader(body.ExecutionPayloadHeader),
 	}
 }
+
+// CopySignedBeaconBlockWithBlobKZGs copies the provided SignedBeaconBlockBellatrix.
```

Comment:
Updated comment (Bellatrix -> WithBlobKZGs) here and for functions below it.

Reply:
Nice catch.

Reply:
fixed. Thanks!